scripts/restore

```diff
@@ -42,8 +42,6 @@ datadir=$(ynh_app_setting_get --app=$app --key=datadir)
 #=================================================
 ynh_script_progression --message="Validating restoration parameters..."
 
-ynh_webpath_available --domain=$domain --path_url=$path_url \
-	|| ynh_die --message="Path not available: ${domain}${path_url}"
```

Comment:
Why this removal?

Reply:
Not needed -> https://github.com/YunoHost/example_ynh/pull/171

Reply:
I see, thanks! :+1: